### ISSUE TITLE
lock landscape mode while taking sequential photos

### ISSUE THREAD

**Issue by call-me-matt** [nobelization/panoramax-mobile-app#99]
I would appreciate a rotation lock once the user starts taking sequential pictures.

when riding my bike, small vibrations make my phone (pixel 8) constantly rotate the screen. of a sequence, about 1/3 of the pictures are rotated or upside down, so I delete them afterwards.

I have deactivated screen rotation in Android. Other apps do not have this issue. The mapillary app rotates only after changing rotation for a second or longer.

**Comment by JLZIMMERMANN** [comment]
Yes here you have some sequences ![Screenshot_2024-10-05-12-37-05-533_app.panoramax.beta.jpg](https://github.com/user-attachments/assets/f01934e2-9c96-4e02-8699-bde7e7f8ec54)
![Screenshot_2024-10-05-12-37-22-083_app.panoramax.beta.jpg](https://github.com/user-attachments/assets/69c2dd9b-c09d-45db-8f28-a5213010acd7)

**Comment by chrisdebian** [comment]
This would be really useful. When I drive on roads with potholes (most roads in UK), this changes the orientation. Locking in 'landscape' would drastically improve the quality and quantity of images I could submit.

Chris
